app/controllers/hyrax/admin/analytics/collection_reports_controller_decorator.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# OVERRIDE: Hyrax hyrax-v3-5-0 file to create and use #present_collections so that we stop showing deleted collections
+# on the collections analytics page
+
+module Hyrax
+  module Admin
+    module Analytics
+      module CollectionReportsControllerDecorator
```

Comment:
module CollectionReportsControllerClassDecorator ??

Reply:
rob said that using “Class” (as in CollectionReportsControllerClassDecorator) isn’t because we’re overriding a class. It has to do with overriding class methods. so because we didn’t have a “self.index” or anything like that, that’s why it looked like the decorator wasn’t being called.